frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -526,7 +526,7 @@ class Worksheet extends React.Component {
         } else {
             // If no location for the insertion is specified,
             // insert the new item at the top of the worksheet in default.
-            actualData['after_sort_key'] = -1;
+            actualData['after_sort_key'] = 0;
```

Comment:
Why change this from -1 to 0?

Reply:
Across the whole codebase, there exists inconsistency for the default after_sort_key previously. So now make them uniform and set it to be 0.